go.mod

```diff
@@ -1,4 +1,3 @@
-module github.com/tlentz/d2modmaker
+module diablomod
```

Comment:
why did this change?

Reply:
I didn't realise that was the mod name. It can be changed back to 'github.com/tlentz/d2modmaker', fair enough.